components/02-elements/tab/tab.js

```diff
@@ -1,14 +1,17 @@
-'use strict'
+'use strict';
 
 const tab                = document.querySelectorAll('.tab__title'),
       content            = document.querySelectorAll('.tab__content'),
       activeTitleClass   = 'tab__title--active',
       activeContentClass = 'tab__content--active';
 
-
 tab.forEach(key => key.addEventListener('click', event => {
-  const self = event.target,
-        tabId = self.dataset.tab;
+  let self = event.target;
+
+  if (self.tagName === 'svg' || self.classList.contains('tab__icon')) {
+    self = self.closest('.tab__title');
```

Comment:
Use `event.currentTarget` instead just `target` and you will be able to remove this code.

Code sample - https://codepen.io/Igloczek/pen/ooxvGQ

Reply:
great ! thanks ;)